packages/mongodb-chatbot-server/src/services/ConversationsService.ts

```diff
@@ -32,6 +32,14 @@ export type AssistantMessage = MessageBase & {
    */
   rating?: boolean;
 
+  /**
+    An additional text comment provided by the user to clarify their
+    `rating` for the message. If `rating` is undefined, this should be
+    too. Note that only messages with `role: "assistant"` can be rated
+    and commented on.
```

Comment:
```suggestion
    too.
```

nit: not nec here b/c this is only on assistant msg type